docs/core/extensions/snippets/logging/getting-started-logger-message/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.Logging;
+
+internal partial class Program
+{
+    static void Main(string[] args)
+    {
+        using ILoggerFactory factory = LoggerFactory.Create(builder => builder.AddConsole());
+        ILogger logger = factory.CreateLogger("Program");
```

Comment:
Why not use `factory.CreateLogger<Program>();`?

Reply:
I was trying to avoid adding another concept I'd need to explain in the introductory sample. As-is we can describe category as a string associated with the ILogger and since the parameter type is a string I expect everyone to understand. If instead we use factory.CreateLogger<Program>() then the explanation of how the type parameter maps to a string has to be front-loaded.